reporting/src/request/index.js

```diff
@@ -350,10 +338,11 @@ export default class RequestReporter {
     if (this.tokenExaminer.examineTokens(state) === false) {
       return response.toWebRequestResponse();
     }
-    // tokenTelemetry.extractKeyTokens
-    delayOneMs(() => {
+
+    queueMicrotask(() => {
```

Comment:
Note that it will also block the event queue, so there is no benefit in running it directly. It will still block the browser, for instance. What it achieves is that it will not be executed directly, but it will first clear the stack, and then run it (after after microtasks queued before, but before giving back control to the browser).

setTimeout(..., 0) would queue it for the next event loop iteration (clearing all microtask and giving back control to the browser in-between).

IMHO, here you could just run it directly.

Reply:
fixed